client/src/actions/index.ts

```diff
@@ -209,6 +209,7 @@ const doInitialDataLoad = (): ((
         isCellGuideCxg,
       });
 
+      // @ts-expect-error (seve): fix downstream lint errors as a result of detailed app store typing
```

Comment:
there's a lot of downstream annomatrix type errors due to the possibility of it being undefined. We may want to write a ticket for this, but the work would be pretty low prio.